bindata/network/openshift-sdn/003-rbac-controller.yaml

```diff
@@ -5,6 +5,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: openshift-infra
```

Comment:
So this isn't actually our Namespace; sdn-controller ends up referring to it because of code it shares with openshift-controller-manager, and no one else has created it yet when we bring up the network, so we have to create it ourselves. We don't create any pods in it though. I don't know if anyone else does. Anyway, whatever change gets made here needs to be coordinate with whatever other module would have been responsible for creating this Namespace if we didn't. (Origin? Installer?)